Source/WebCore/dom/FragmentDirectiveRangeFinder.cpp

```diff
@@ -206,7 +209,7 @@ static std::optional<SimpleRange> rangeOfStringInRange(const String& query, Simp
         RefPtr<Node> currentNode = &searchRange.startContainer();
         
         if (isNonSearchableSubtree(*currentNode)) {
-            if (auto newStart = NodeTraversal::nextSkippingChildren(*currentNode))
+            if (auto newStart = nextSkippingChildrenInComposedTreeIgnoringUserAgentShadow(*currentNode))
```

Comment:
Composed tree means flat tree, it's different from [shadow-including tree order](https://dom.spec.whatwg.org/#concept-shadow-including-tree-order).

Reply:
Thanks, that naming is always confusing to me. Anyway, I removed that part since it is not needed to fix the actual test I am trying to fix.